### PR TITLE
fixes insert step functionality

### LINKED ISSUE #1261: bugs in insert step functionality
- [ ] insert steps add step multiple number of time and are mostly added after some random step.

- [ ] close button and radio group buttons are not working

- [ ] there are some undefined options in select module dropdown
Sep 23, 2019 8_49 PM (2)

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by keshav234156**
Fixes #1261 
20191001165455-4fcb4dbd00 gif-2-mp4 com

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below
* [ ] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by keshav234156** [comment]
@jywarren @HarshKhandeparkar  can you please review this PR!!!

**Comment by HarshKhandeparkar** [comment]
Do you know why this is happening?
insert-bug

**Comment by keshav234156** [comment]
@HarshKhandeparkar could you please tell the step to reproduce the above

**Comment by HarshKhandeparkar** [comment]
Add a few steps, insert any step and try collapsing the dropdowns.

**Comment by keshav234156** [comment]
there might be some error here that is causing this behavior
https://github.com/publiclab/image-sequencer/blob/44f5cb4e03a48973953f93f715aa188d499b6df5/examples/lib/defaultHtmlStepUi.js#L192

This behaviour is shown in beta version as well.So don't think that it has been caused by this PR.There are lots of other error in examples/lib/defaultHtmlStepUi.js,Will open a issue for  same

**Comment by HarshKhandeparkar** [comment]
ok. ty.

**Comment by keshav234156** [comment]
@jywarren can you please review this PR!!!

File: examples/lib/intermediateHtmlStepUi.js [hunk 1]
```diff
@@ -104,61 +95,43 @@ function IntermediateHtmlStepUi(_sequencer, step, options) {
...
   };
 
-  function insert(id, $step) {
-
-    options = options || {};
-    var insertStepSelect = $step('.insert-step-select');
-    if (insertStepSelect.val() == 'none') return;
-
-    var newStepName = insertStepSelect.val();
+  function insert(id, $step, newStepName) {
```
Comment (jywarren) [inline examples/lib/intermediateHtmlStepUi.js]: This is looking good -- do you think we could make the name here more specific? Like, insert what? Insert step, right? 

Thank you!!!

Comment (keshav234156) [reply examples/lib/intermediateHtmlStepUi.js]: @jywarren was thinking the same but insertstep was already defined in line 77,so left as it is

**Comment by jywarren** [comment]
Thanks for the review @vaibhavmatta have you tested this locally and confirmed the fix? Thanks all!

**Comment by vaibhavmatta** [comment]
@jywarren Yes!! Though I tested it for the first time, but it worked fine for me. THANKS

**Comment by jywarren** [comment]
Great, thanks all!!!!!

**Pull request merged**